src/System.Threading.Tasks.Dataflow/src/Base/DataflowBlock.cs

```diff
@@ -15,12 +15,8 @@
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using System.Diagnostics.Contracts;
-using System.Linq;
 using System.Runtime.CompilerServices;
-using System.Runtime.ExceptionServices;
-using System.Security;
 using System.Threading.Tasks.Dataflow.Internal;
-using System.Threading.Tasks.Dataflow.Internal.Threading;
```

Comment:
@safern @stephentoub Thanks for fixing it.

Would it be worth ensuring that some future ReSharper-trigger-happy contributor like me doesn't break it again? One way would be to declare the `THREADING` symbol for the .Net Standard 2.0 configuration and then doing the following in the two files that use it:

```c#
#if !THREADING
using System.Threading.Tasks.Dataflow.Internal.Threading;
#endif
```

As an added advantage, it would mean the .Net Standard 2.0 version of the package wouldn't contain the unused `Internal.Threading` types.